client/ayon_core/hosts/hiero/api/plugin.py

```diff
@@ -771,47 +790,44 @@ def _populate_attributes(self):
         log.debug(
             "____ self.shot_num: {}".format(self.shot_num))
 
+       # publisher ui attribute inputs or default values if gui was not used
+        def get(key):
+            """Shorthand access for code readability"""
+            return self.pre_create_data.get(key)
+
         # ui_inputs data or default values if gui was not used
-        self.rename = self.ui_inputs.get(
-            "clipRename", {}).get("value") or self.rename_default
-        self.clip_name = self.ui_inputs.get(
-            "clipName", {}).get("value") or self.clip_name_default
-        self.hierarchy = self.ui_inputs.get(
-            "hierarchy", {}).get("value") or self.hierarchy_default
-        self.hierarchy_data = self.ui_inputs.get(
-            "hierarchyData", {}).get("value") or \
-            self.track_item_default_data.copy()
-        self.count_from = self.ui_inputs.get(
-            "countFrom", {}).get("value") or self.count_from_default
-        self.count_steps = self.ui_inputs.get(
-            "countSteps", {}).get("value") or self.count_steps_default
-        self.subset_name = self.ui_inputs.get(
-            "subsetName", {}).get("value") or self.subset_name_default
-        self.subset_family = self.ui_inputs.get(
-            "subsetFamily", {}).get("value") or self.subset_family_default
-        self.vertical_sync = self.ui_inputs.get(
-            "vSyncOn", {}).get("value") or self.vertical_sync_default
-        self.driving_layer = self.ui_inputs.get(
-            "vSyncTrack", {}).get("value") or self.driving_layer_default
-        self.review_track = self.ui_inputs.get(
-            "reviewTrack", {}).get("value") or self.review_track_default
-        self.audio = self.ui_inputs.get(
-            "audio", {}).get("value") or False
+        self.rename = get("clipRename") or self.rename_default
```

Comment:
Why not use this instead? Expression `or` might be dangerous considering there might be value whith returns `False`, like `0`, `False`, `[]`, `{}`...
```suggestion
        self.rename = self.pre_create_data.get("clipRename", self.rename_default)
```

Reply:
This was marked as resolved, but nothing changed in code.